Modules/sha3module.c

```diff
@@ -639,8 +639,10 @@ _sha3_exec(PyObject *m)
     init_sha3type(shake_256_type, SHAKE256_spec);
 #undef init_sha3type
 
-    if (PyModule_AddStringConstant(m, "implementation",
-                                   "HACL") < 0) {
+    if (PyModule_AddStringConstant(m, "implementation", "HACL") < 0) {
+        return -1;
+    }
+    if (PyModule_AddIntConstant(m, "GIL_MINSIZE", HASHLIB_GIL_MINSIZE) < 0) {
```

Comment:
in all of these extension modules I suggest exposing this as _GIL_MINSIZE with the _ prefix just in case to avoid anyone happening to think of it as an attribute to rely on.  (even though these are supposed to be internal implementation detail modules).